parsing/parser.mly

```diff
@@ -1521,16 +1520,16 @@ class_sig_field:
       { Pctf_attribute $1 })
       { $1 }
 ;
-value_type:
+%inline value_type:
     VIRTUAL mutable_flag mkrhs(label) COLON core_type
       { $3, $2, Virtual, $5 }
   | MUTABLE virtual_flag mkrhs(label) COLON core_type
       { $3, Mutable, $2, $5 }
   | mkrhs(label) COLON core_type
       { $1, Immutable, Concrete, $3 }
 ;
-constrain:
-    core_type EQUAL core_type
+%inline constrain:
```

Comment:
I have no idea why the `constrain:` definition is place here, it should be below `constraints:` on line 2343, would you move it there?